### ISSUE TITLE
Export AxiosObservable in main index.ts file

### ISSUE THREAD

**Issue by yf-hk** [zhaosiyang/axios-observable#20]
Hi, thanks for making this wonderful module.
What I'm currently doing is like below:
```ts
jobs$
  .pipe(
    mergeMap<string, AxiosObservable<string>>(
      (url = 'https://www.baidu.com') => {
        return Axios.get(url);
      },
    ),
    map((result) => result.data),
    tap((html) => {
      console.log(html);
    }),
  )
```
So I need to specify the input and output data type of `mergeMap` function in order to get it work. In that case I need to import `AxiosObservable`. Is this the best way to use it? If `AxiosObservable` is needed, can you please export it in the main project file, instead of 'axios-observable/dist/axios-observable.interface'?
Something like this would be fine:
```ts
import { AxiosObservable } from 'axios-observable';
```

**Comment by gabrielthomasjacobs** [comment]
I think you can just specify the type as Observable<string> since AxiosObservable extends Observable.

**Comment by zhaosiyang** [comment]
@dzcpy Thank you for your issue. I've fixed it in the latest version 1.2.0
